code/lib/create-storybook/src/ink/procedures/Telemetry.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { useContext, useEffect, useState } from 'react';
+
+import { Box, Text } from 'ink';
+
+import type { State } from '../steps';
+import { AppContext } from '../utils/context';
+
+export function Telemetry({ state }: { state: State }) {
+  const { telemetry } = useContext(AppContext);
+
+  useEffect(() => {
+    const { intents, framework, features, version, install, directory } = state;
+    telemetry?.('init', { intents, framework, features, version, install, directory });
```

Comment:
`directory` will be an absolute path, this is considered sensitive information, we should sanitize it.

Reply:
Yeah, do you think we should just omit it?